hstrat/dataframe/surface_unpack_reconstruct.py

```diff
@@ -0,0 +1,143 @@
+import argparse
+import logging
+
+from joinem._dataframe_cli import _add_parser_base, _run_dataframe_cli
+
+from .._auxiliary_lib import get_hstrat_version
+from ._surface_unpack_reconstruct import surface_unpack_reconstruct
+
+
+def lws(line):
+    prefix = line[: len(line) - len(line.lstrip())]
+    return len(prefix) + 3 * prefix.count("\t")
+
+
+class SmartHelpFormatter(argparse.HelpFormatter):
```

Comment:
maybe this should go in auxlib? and needs a docstring and a descriptive name (I assume this has something to do with reflowing text?) Should also have type hints